aws-lambda-scorer-python/lambda-template/py_scorer.py

```diff
@@ -0,0 +1,120 @@
+import os
+import json
+import boto3
+import botocore
+import daimojo.model
+import datatable as dt
+
+s3 = boto3.resource('s3')
+
+
+class ScorerError(Exception):
+    """Base Scorer Error"""
+    status_code = 500
+
+
+class InternalError(ScorerError):
+    """Internal Error"""
+    status_code = 500
+
+
+class BadRequest(ScorerError):
+    """Bad request"""
+    status_code = 400
+
+
+class NotFound(ScorerError):
+    """Not found"""
+    status_code = 404
+
+
+def get_object_from_s3(bucket_name, bucket_key):
+    try:
+        s3.Bucket(bucket_name).download_file(bucket_key, '/tmp/pipeline.mojo')
```

Comment:
I think we should create a memory reader for the cpp runtime to not have to rely on storing mojo on disk

Reply:
That can be added. Can you open an issue in mojo2 repo?

I will work on this after we release 1.7.2.

Reply:
https://github.com/h2oai/mojo2/issues/736